templates/search/clustering_facet.html

```diff
@@ -0,0 +1,8 @@
+<ul>
+    {% for cluster_id, num_results, tags in cluster_id_num_results %}
+        <li class="facet_item">
+            <a href=".?{{url_query_params_string}}&cluster_id={{cluster_id|add:1}}">Cluster #{{cluster_id|add:1}}
```

Comment:
By including this here, I have a bug that makes the url to grow everytime a clustering facet filter is applied.